src/styles/components/_component-grid.scss

```diff
@@ -0,0 +1,83 @@
+$breakpoint-tablet: 820px;
+
+.components {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 1rem;
+
+    &-grid {
+    width: 100%;
+    padding: 0;
+  }
+  &-grid__item {
+    width: 100%;
+  }
+
+  &-grid {
+    &__item {
+      display: flex;
+      justify-content: center;
```

Comment:
Could we also add `align-items: center` to the `.components-grid__item` (about line 20) that way the images don't elongate when the screen is an "in-between" size? The accordion and the dialog box in the screenshot below are a little funky. 
<img width="806" alt="Screen Shot 2022-10-17 at 9 10 24 AM" src="https://user-images.githubusercontent.com/69602589/196186221-d0655fca-8d46-4168-9755-aa0903183a76.png">